### ISSUE TITLE
PoolNumber is overriding deprecated methods on an instance of NumberEntity

### ISSUE THREAD

**Issue by BillyFKidney** [jlvaillant/intellicenter#52]
I'm receiving the following warning upon boot of my Home Assistant instance

- Home Assistant 2022.7.7
- Supervisor 2022.07.0
- Operating System 8.4
- Frontend 20220707.1 - latest

```
Logger: homeassistant.components.number
Source: components/number/__init__.py:220 
Integration: Number (documentation, issues) 
First occurred: 2:02:55 PM (1 occurrences) 
Last logged: 2:02:55 PM

custom_components.intellicenter.number::PoolNumber is overriding deprecated methods on an instance of NumberEntity, this is not valid and will be unsupported from Home Assistant 2022.10. Please report it to the custom component author.
```

**Comment by dwradcliffe** [comment]
I have a fix for this, will push it later this week.

**Comment by cymone-smart** [comment]
Getting same message.

**Comment by dwradcliffe** [comment]
Fixed in #55

**Comment by thewolfman56** [comment]
Does anyone have Home Assistant 2022.10 working with this integration?  I'm still on 2022.9.7 because this is one of the most frequently used integrations for us and I cannot risk not having it available.

**Comment by TheRydad** [comment]
I'm on 2022.10.4. I do see the unsupported warning message in my log, but everything appears to be working fine with the integration. I guess I wasn't paying attention.

I am hesitant to install dwradcliffe's fork, but might look into it if it isn't merged and breaks. I don't think I have a reason to go 11 yet.

**Comment by BillyFKidney** [comment]
It works on 2022.10 and 2022.11; you’ll just receive a warning. - Billy -On Nov 14, 2022, at 10:11 AM, thewolfman56 ***@***.***> wrote:﻿
Does anyone have Home Assistant 2022.10 working with this integration?  I'm still on 2022.9.7 because this is one of the most frequently used integrations for us and I cannot risk not having it available.

**Comment by thewolfman56** [comment]
> I'm on 2022.10.4. I do see the unsupported warning message in my log, but everything appears to be working fine with the integration. I guess I wasn't paying attention.
> 
> I am hesitant to install dwradcliffe's fork, but might look into it if it isn't merged and something breaks with another update. I don't think I have a reason to go 11 yet.

Thanks.  I just looked into that fork and the commits look like they address this issue and may address an issue I had with circuit groups (but used scenes and switch templates in HA to work around that limitation).  I may switch over though if this one starts to appear abandoned.

**Comment by thewolfman56** [comment]
> It works on 2022.10 and 2022.11; you’ll just receive a warning. - Billy -On Nov 14, 2022, at 10:11 AM, thewolfman56 ***@***.***> wrote:﻿ Does anyone have Home Assistant 2022.10 working with this integration? I'm still on 2022.9.7 because this is one of the most frequently used integrations for us and I cannot risk not having it available. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

Thanks.  There are a couple of changes I wanted to try with 10 and 11 may end up being better, but breaking changes often has me hesitant to upgrade right away.

**Comment by TheRydad** [comment]
The owner, @jlvaillant, had been active in here a few months ago with an issue related to 2022.06. I hope it's not abandoned.

**Comment by thewolfman56** [comment]
> The owner, @jlvaillant, had been active in here a few months ago with an issue related to 2022.06. I hope it's not abandoned.

My concern is it looks like he has not been active in github in a few months and the pull request to correct this new error was made in September, so hoping everything is okay and the main branch can be updated.

**Comment by jlvaillant** [comment]
Yes, sorry guys, personal issues have been keeping me from being on top of things. My apologies…. I’ll try to merge things and produce a good release asap… Sent from my iPhoneOn Nov 17, 2022, at 10:53 PM, thewolfman56 ***@***.***> wrote:﻿

The owner, @jlvaillant, had been active in here a few months ago with an issue related to 2022.06. I hope it's not abandoned.

My concern is it looks like he has not been active in github in a few months and the pull request to correct this new error was made in September, so hoping everything is okay and the main branch can be updated.
